addons/core/tests/integration/helpers/reverse-indexed-display-name-test.js

```diff
@@ -17,26 +17,26 @@ module('Integration | Helper | reverse-indexed-display-name', function (hooks) {
     this.set('array', ['Boundary', 'is', 'cool!']);
 
     await render(hbs`{{reverse-indexed-display-name
-      'resources.session-recording.connection.title_index'
-      this.array
-      'Boundary'
-    }}`);
+  'resources.session-recording.connection.title_index'
```

Comment:
I have noticed we changed indentation in test files, but I do not see any changes on prettier config file, what is the reason behind this change?

Reply:
@calcaide looking into this.